server/routes/caseNotes/caseNotesView.ts

```diff
@@ -0,0 +1,38 @@
+import { TableArgs } from '../../utils/govukFrontendTypes'
+import CaseNotesPresenter from './caseNotesPresenter'
+import ViewUtils from '../../utils/viewUtils'
+
+export default class CaseNotesView {
+  constructor(private presenter: CaseNotesPresenter) {}
+
+  private get tableArgs(): TableArgs {
+    return {
+      head: [
+        { text: 'Details', classes: 'govuk-!-width-one-quarter' },
+        { text: 'Case notes', classes: 'govuk-!-width-three-quarters' },
+      ],
+      rows: this.presenter.tableRows.map(row => {
+        return [
+          {
+            html: `<p class="govuk-body">${row.sentAtDay}<br>${row.sentAtDate}<br>${row.sentAtTime}<br>${row.sentBy}</p>`,
+          },
+          {
+            html: `<p class="govuk-body"><b>${ViewUtils.escape(row.subject)}</b><br><br>${ViewUtils.escape(
```

Comment:
sorry i missed this before - please can we also preserve line breaks here?

`ViewUtils.nl2br(ViewUtils.escape(...))`

Reply:
and this is a small nit, but previously when i mentioned govukstyles i was thinking more about using those styles to format the bold text using one of the heading classes. it's inconsequential, but would be preferred if it doesn't totally screw the formatting.

Reply:
Thanks, I missed this.